scripts/components/Main.js

```diff
@@ -44,6 +45,8 @@ export default class Main extends React.Component {
         focusedInteraction: e.data,
       });
     });
+    // Show scene description when scene starts for the first time, if specified
+    this.handleSceneDescriptionInitially(this.props.currentScene);
```

Comment:
```suggestion
	if(!this.context.extras.isEditor) {
    this.handleSceneDescriptionInitially(this.props.currentScene);
    }
```

Just a check so that it doesnt up everytime we enter the editor, since edit mode refreshes the context